.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable/possible, add screenshots to help explain your problem.
```

Comment:
I assume this is to inspire some creativity! You could also mention adding a GIF of the issue (like from ScreenToGIF) if appropriate!

Reply:
e.g. https://github.com/mantidproject/mantid/issues/28306#issuecomment-696756489